public/js/index_datatables.js

```diff
@@ -109,7 +109,7 @@ function createNamespaceColumn(namespace, type, data) {
 		match = regex.exec(data);
 
 		if (match != null) {
-			return '<a style="cursor:pointer" onclick="$(\'#srch\').val($(this).html()); arcTable.search($(this).html()).draw();">' +
+			return '<a style="cursor:pointer" arc-namespace="' + key + '" onclick="$(\'#srch\').val($(this).attr(\'arc-namespace\') + \':\' + $(this).html()); arcTable.search($(this).attr(\'arc-namespace\') + \':\' + $(this).html()).draw();">' +
```

Comment:
there's no `key` variable here, you'll need to use `namespace`. Be careful tho that namespace can be modified to the `(?:series|parody)` regex on line 107, so you'll need to copy that to another variable.